tests/bud.bats

```diff
@@ -693,3 +693,49 @@ load helpers
   buildah rm ${cid}
   buildah rmi ${target}
 }
+
+@test "bud-onbuild" {
+  target=onbuild
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/onbuild
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild1" "RUN touch /onbuild2"]' ]
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  run ls ${root}/onbuild1 ${root}/onbuild2
+  echo "$output"
+  [ "$status" -eq 0 ]
+  buildah umount ${cid}
+  buildah rm ${cid}
+
+  target=onbuild-image2
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} -f Dockerfile1 ${TESTSDIR}/bud/onbuild
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild3"]' ]
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  run ls ${root}/onbuild1 ${root}/onbuild2
+  echo "$output"
+  [ "$status" -eq 0 ]
+  run ls ${root}/onbuild3
+  echo "$output"
+  [ "$status" -ne 0 ]
```

Comment:
Redhat CI is staying that line 725 failed.

Reply:
running the test by hand, I think line 725 s/b '-eq 0' as the directory is there.

Reply:
Yes the buildah from changes are causing the tests to blow up.  Fixing up tests.